src/fe-common/core/fe-common-core.c

```diff
@@ -470,16 +470,23 @@ gboolean strarray_find_dest(char **array, const TEXT_DEST_REC *dest)
 		return TRUE;
 
 	if (dest->server_tag != NULL) {
-		char *tagtarget = g_strdup_printf("%s/%s", dest->server_tag, "*");
+		char *tagtarget = g_strdup_printf("%s/%s", dest->server_tag, "#");
```

Comment:
You'll have to explain the git steps one by one :) Close the PR? revert? I do not know!